docs/editor/workspaces.md

```diff
@@ -53,9 +53,11 @@ You don't have to do anything for a folder to become a VS Code workspace other t
 
 ![Multi-root workspace](images/workspaces/multi-root-workspace.png)
 
+Multi-root workspaces are very useful to implement the loose definition of "project" stated above.
+
 *A multi-root workspace opened in VS Code*
 
-> **Note:** The visual difference of having a folder opened versus opening a `.code-workspace` file can be subtle. To give you a hint that a `.code-workspace` file has been opened, some areas of the user interface (for example, the root of the File Explorer) show an extra **(Workspace)** suffix next to the name.
+> **Note:** The visual difference of having a folder opened versus opening a `.code-workspace` file can be subtle. To give you a hint that a `.code-workspace` file has been opened, some areas of the user interface (for example, the root of the File Explorer) show an extra **(WORKSPACE)** suffix next to the name.
```

Comment:
In the window title the suffix is mixed case. Also in the recently-used lists. I think the change here should be reverted.